### PR TITLE
feat: set background image and wallet status bar

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes # mf-702

1、set background image (mf-702）
https://mask.atlassian.net/jira/software/c/projects/MF/boards/25?modal=detail&selectedIssue=MF-702
2、modify avatar meta data (mf-702)

### CODE REVIEW

File: packages/mask/shared/kv-storage.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const InMemoryStorages = {
         schema: 1,
         chainId: 1,
         pluginId: '',
+        flag: 0,
```
Comment (UncleBill) [inline packages/mask/shared/kv-storage.ts]: `flag` is not clear enough.

File: packages/mask/src/plugins/Avatar/Application/UploadAvatarDialog.tsx [hunk 2]
```diff
@@ -10,10 +10,9 @@ import type { BindingProof } from '@masknet/shared-base'
 import { useI18N } from '../locales/i18n_generated'
 import { context } from '../context'
 import { useSubscription } from 'use-subscription'
-import type { NetworkPluginID } from '@masknet/web3-shared-base'
 import { useCurrentWeb3NetworkPluginID } from '@masknet/plugin-infra/web3'
 import { AvatarInfo, useSave } from '../hooks/save/useSave'
-import type { AllChainsNonFungibleToken } from '../types'
+import { SelectTokenInfo, SET_NFT_FLAG } from '../types'
```
Comment (UncleBill) [inline packages/mask/src/plugins/Avatar/Application/UploadAvatarDialog.tsx]: `SET_NFT_FLAG` is not clear enough. I see that it's used across multiple files, but I can't understand it by its name.

File: packages/mask/src/plugins/Avatar/types.ts [hunk 3]
```diff
@@ -51,3 +53,10 @@ export interface NFTInfo {
     slug: string
     permalink: string
 }
+
+export enum SET_NFT_FLAG {
```
Comment (UncleBill) [inline packages/mask/src/plugins/Avatar/types.ts]: I would suggest
```suggestion
export enum NFT_USAGE {
```

or

```suggestion
export enum NFT_SHOWCASE {
```

File: packages/shared/src/UI/components/WalletStatusBar/WalletBarButton.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import type { WalletButtonActionProps } from './types'
+
+interface WalletButtonProps extends withClasses<'root'> {
+    actionProps?: WalletButtonActionProps
+    onConnectWallet?: () => void
+}
+
+const useStyles = makeStyles<{ color?: 'warning' }>()((theme, props) => ({
+    progress: {
+        color: MaskColorVar.twitterButtonText,
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/WalletStatusBar/WalletBarButton.tsx:16]: We don't use `MaskColorVar` anymore. Use `theme.palette.maskColor` instead. You can consult @nuanyang233 for more information.

**Comment by lelenei** [comment]
don't do it

**Pull request closed without merging**